### ISSUE TITLE
lslint fails (Using wrong folder)

### ISSUE THREAD

**Issue by WolfGangS** [buildersbrewery/sublime-lsl#7]
## Tools > Build Fails
sublime-lsl is looking in the wrong folder for included lslint executable

it is looking inside the `bin` folder, the files are in the `lib` folder.

### Expected behavior
pressing tools > build in the menu should not error with failure to find file

### Actual behavior
Error thrown in console
```
[WinError 2] The system cannot find the file specified
[cmd: ['C:\\Users\\user\\AppData\\Roaming\\Sublime Text 3\\Packages\\=BB= LSL\\bin\\lslint\\windows\\lslint.exe', '-i', '-m', '-p', '-w', '-z', 'C:\\Users\\user\\projects\\sl\\lsl\\file.lsl']]
[dir: C:\Users\user\projects\sl\lsl]
```

### Steps to reproduce
Open lsl file, press the build option in the tools menu and build for any windows

### Environment

```text
=BB= LSL:
    version: 4.3.0
    installed via Package Control: True

Sublime Text:
    channel: dev
    build: 3189
    portable: True
    platform: Windows
    architecture: x64

Packages:
    Package Control: v3.3.0
    PackageDev: v3.2.7

Dependencies:
    SublimeLinter: 4.9.4
    markupsafe: 0.23.0
    mdpopups: 3.4.0
    pygments: 2.0.2
    python-jinja2: 2.8.0
    python-markdown: 2.6.11
    pyyaml: 3.12.0
    ruamel-yaml: 1.2.0
```

**Comment by buildersbrewery** [comment]
Fixed in v4.3.5, thanks for the report. Will push a fix within the hour.

**Comment by buildersbrewery** [comment]
@WolfGangS Please re-open if problems persist.

---

You might want to delete cache after updating and closing the app, before re-opening.

<kbd>Main menu > View > Show Console</kbd>

`import sublime, webbrowser; webbrowser.open('file:///' + sublime.cache_path())`
